README.md

```diff
@@ -1,2 +1,167 @@
-# Capstone-Project_sam
-This is capstone project.
+# Capstone-Project
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Capstone-Project <a name="about-project"></a>
+
+**Capstone-Project** is developed in HTML, CSS, JavaScript, and Bootstrap framework and is about the innovative summit of Techlife development company which provides various information about Techlife. This project also showcases my skills as a developer.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>
+      HTML <br>
+      CSS
+      </a>
+    </li>
+  </ul>
+</details>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+features
+- [Live Demo link](https://github.com/samuelkinyanjui/Capstone-Project_sam)
```

Comment:
[ ] The link provided isn't working, make sure to deploy your project using GitHub pages and then you can add the link. Kindly fix that 🤓. Look at this [article](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) it might help you.